apps/src/templates/projects/ProjectAppTypeArea.jsx

```diff
@@ -109,17 +110,19 @@ class ProjectAppTypeArea extends React.Component {
    *   completed and the done handler has been run (if successful).
    */
   fetchOlderProjects() {
-    const {projectList, labKey: projectType} = this.props;
+    const projectType =
+      this.props.labKey === 'featured' ? 'all' : this.props.labKey;
+    const {projectList} = this.props;
     const oldestProject = projectList[projectList.length - 1];
-    const oldestPublishedAt =
-      oldestProject && oldestProject.projectData.publishedAt;
+    const oldestFeaturedAt =
```

Comment:
Just looking at prop types, and `featuredAt` doesn't look like it's listed as available in what gets passed to this component...might just need to update the type?